spec/helpers/produce_items_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProduceItemsHelper, type: :helper do
+  context 'when viewing the produce items index' do
+    let!(:produce_item) { create(:produce_item) }
+    let!(:produce_item_two) { create(:produce_item) }
+
+    describe '#formatted_month_name' do
+      before do
+        create(:item_season, produce_item: produce_item, month_index: 1)
+        create(:item_season, produce_item: produce_item, month_index: 2)
+        create(:item_season, produce_item: produce_item_two, month_index: 3)
+      end
+
+      it 'adds / to the string, if its not the last or only month' do
```

Comment:
**suggestion:** `if its not the last or only month` reads a bit weird to me. This is a bit too close to the implementation. You could say something like `formats months using / as a separator`. The fact that your expected string contains two months separated by `/` is already indication that things are working as expected. Does that make sense to you?

Reply:
yes! completely agree 🙂